media/js/base/mozilla-monitor-button.js

```diff
@@ -12,27 +12,28 @@ if (typeof window.Mozilla === 'undefined') {
 
     var MonitorButton = {};
 
-    MonitorButton.init = function(buttonId){
+    MonitorButton.init = function(){
+        var buttons;
 
-        // check for custom buttonId, used in case of multiple buttons.
-        if(!buttonId){
-            buttonId = 'fxa-monitor-submit';
-        }
+        // Collect all monitor buttons
+        buttons = document.getElementsByClassName('js-monitor-button');
 
-        var monitorButton = document.getElementById(buttonId);
+        // Exit if no valid button in DOM
+        if (buttons[0] === undefined) {
```

Comment:
Use `length` property?
```suggestion
        if (buttons.length === 0) {
```